static/app/views/settings/organization/organizationSettingsNavigation.tsx

```diff
@@ -19,27 +17,32 @@ type State = {
   hooks: React.ReactElement[];
 };
 
-const OrganizationSettingsNavigation = createReactClass<Props, State>({
-  displayName: 'OrganizationSettingsNavigation',
+class OrganizationSettingsNavigation extends React.Component<Props, State> {
+  state = this.getHooks();
```

Comment:
needs `state: State = this.getHooks();` or `getHooks(): State` or the component will use the object returned from getHooks as the state